ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1452,6 +1452,18 @@ FCU_DFSS_DSP_HWZC_CHWZC_CO2M:
   - HWZC
   - CHWZC
   - CO2M
+
+FCU_RHDHC_HTRC_DXRC_ZTM:
```

Comment:
Where are the fan status points?

Reply:
This is the crac unit with no fan points available in the bms.